brands/shop/supermarket.json

```diff
@@ -649,6 +649,7 @@
     }
   },
   "shop/supermarket|Co-op": {
+    "countryCodes":["ca"],
```

Comment:
BTW: there is also a supermarket called `Coop` in some european countries: https://en.wikipedia.org/wiki/Coop_(Switzerland)
This probably deserves an extra entry because it is a different brand than this one...

Reply:
hmm I see. Okay I will add them soon. However, I need to check there are how many Coop, in fact, first.

Reply:
I added this in 75452f4 and added disambiguation text and resolved all the remaining warnings.

Now we support:
      "amenity/fuel|Co-op~(Canada)",
      "shop/convenience|Coop~(Schweiz)",
      "shop/convenience|The Co-operative Food~(UK)",
      "shop/supermarket|Co-op~(Canada)",
      "shop/supermarket|Coop~(Schweiz)",
      "shop/supermarket|The Co-operative Food~(UK)"